Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/rolling_paper.yml

```diff
@@ -4,10 +4,15 @@
   id: PackPaperRolling
   description: "A pack of thin pieces of paper used to make fine smokeables."
   components:
-    # I know but it just works.
-  - type: FoodContainer
-    prototypes:
-      PaperRolling: 100
+  - type: Storage
+    whitelist:
+      tags:
+      - RollingPaper
+    capacity: 16
+  - type: StorageFill
+    contents:
+      - id: PaperRolling
```

Comment:
rollingpaper paperrolling rollingpaper paperrolling

Reply:
Should be PaperRolling